### ISSUE TITLE
Field in form 'name' and 'email' on tablet not clickable

### ISSUE THREAD

**Issue by sibsj** [tranduyhung/grav-theme-mache#19]
Hi, it's not possible to enter anything in the fields 'name' and 'email' in the form when using a tablet (tested on ipad). Does anyone have an advice how to solve this issue? Thank you

**Comment by tranduyhung** [comment]
I don't see this problem on iPad. Does it happen on https://grav-theme-mache.flatfiledeveloper.com/ for you?

**Comment by sibsj** [comment]
Yes, it's happening there aswell. You don't have the same issue?

**Comment by tranduyhung** [comment]
Yes, I don't see any problems on my iPad and mobile phones.
